apps/api/src/bank-transactions-file/helpers/parser.ts

```diff
@@ -39,3 +46,11 @@ export function parseBankTransactionsFile(
   })
   return accountMovements
 }
+
+function isBillingNamePresent(movementFunctionalType: string, billingName: string) {
+  return !(
```

Comment:
It's really tricky to see the reverse sign `!` here

I suggest we take the `not` outside of the function

```tsx
if (!isBillingNamePresent()) {

}
```

We can rename the function to state its purpose better if we need to.

Some explanation as comments would be useful here too